docs/en/changelog.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## v0.14.0 (01/04/2022)
+
+**Highlights**
+
+1. Support AOT-GAN
```

Comment:
AOT-GAN is not completely merged and some docs are note ready. Should we move it to v0.15?